client/core/types.go

```diff
@@ -170,7 +173,10 @@ type PostBondForm struct {
 	Asset    *uint32          `json:"assetID,omitempty"` // do not default to 0
 	Bond     uint64           `json:"bond"`
```

Comment:
I kinda wish this was `Tier uint64` instead of `Bond uint64`. I suppose using a bond amount allows us to do some validation that might catch user errors, but specifying (tier, asset) seems easier. The UI will use tier as an input (e.g. `bondStrengthField` in #2025), and it would be better via RPC too, but easier to mess up.

Reply:
This is where the `bondopt` RPC (and `UpdateBondOptions` method would apply).  As noted in your previous comment, postbond should really just be used to register.

Reply:
> It looks like the new `rotateBonds` method will use the server's config value as is. Should it check back with this original value? (Maybe it does and I missed it.)

Only indirectly via the `MaxBondedAmt` set, but I think I've broken this just recently.